src/NuGet.Services.EndToEnd/Support/AssemblyMetadataPackageFile.cs

```diff
@@ -28,6 +29,8 @@ public AssemblyMetadataPackageFile(string resourceName)
 
         public FrameworkName TargetFramework { get; set; }
 
+        public NuGetFramework NuGetFramework { get; set; }
```

Comment:
why is this needed?

Reply:
The `IPackageFile` interface changed via the NuGet client package update.